iot/manager/manager.go

```diff
@@ -24,12 +24,10 @@ import (
 	// [END imports]
 )
 
-// Registry Management
-
-// [START iot_create_registry]
+// [START get_client]
 
-// createRegistry creates a device registry.
-func createRegistry(w io.Writer, projectID string, region string, registryID string, topicName string) (*cloudiot.DeviceRegistry, error) {
+// getClient returns a client based on the environment variable GOOGLE_APPLICATION_CREDENTIALS
+func getClient() (*cloudiot.Service, error) {
```

Comment:
It leads to a lot of duplication, but I think it's better to initialize the client directly in each sample. It can be confusing to go back and forth.